public/app/core/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx

```diff
@@ -135,7 +135,7 @@ const DraggableFieldName: React.FC<DraggableFieldProps> = ({
         >
           <div className="gf-form gf-form--grow">
             <div className="gf-form-label gf-form-label--justify-left width-30">
-              <i className={cx('fa fa-ellipsis-v', styles.draggable)} />
+              <Icon name="draggabledots" size="xl" className={styles.draggable} />
               <IconButton
```

Comment:
Based on discussion, I would remove the padding and change to `size="lg"` for consistent sizing.